shishito/runtime/environment/node_webkit.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
+
+from selenium import webdriver
+import os
+import sys
+from shishito.runtime.environment.shishito import ShishitoEnvironment
+
+
+class ControlEnvironment(ShishitoEnvironment):
+    """ Node-webkit control environment. """
+
+    def call_browser(self, config_section):
+        """ Start chromedriver for node-webkit application. Chromedriver have to be located in same
+        folder as node-webkit application (AUT).
+
+        :param str config_section: section in platform/environment.properties config
+        :return: created webdriver
+        """
+        capabilities = ''
```

Comment:
if you indeed don't need to pass any capabilities to `start_driver` method - because it does not use any, then we don't need this variable at all.

Instead, remove the capabilities parameter from `start_driver` completely or at least make it optional: `start_driver(self, node_webkit_path, capabilities=None)`.

That way you don't need to pass any capability parameter to `start_driver`..